### PR TITLE
feat(components): add apps menu to header

### PR DESCRIPTION
**Pull request by jennieramida**

### CODE REVIEW

**Comment by Benzbeeb** [comment]
LGTM

File: .github/workflows/jest.yml [hunk 1]
```diff
@@ -11,10 +11,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: pnpm/action-setup@v2
+      - uses: pnpm/action-setup@v4
```
Comment (evilpeach) [inline .github/workflows/jest.yml:14]: please rebase or merge develop.

File: src/lib/layout/AppMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                    justifyContent="space-between"
+                    w="full"
+                  >
+                    <Image
+                      src={app.logo}
+                      alt={app.name}
+                      height="20px"
+                      minH="20px"
+                    />
+                    {/* <CustomIcon
```
Comment (evilpeach) [inline src/lib/layout/AppMenu.tsx:122]: remove?

File: src/lib/layout/AppMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        <Flex direction="column">
+          {appList.map((app) => (
+            <Flex
+              key={app.name}
+              py={3}
+              px={4}
+              borderRadius={8}
+              w="full"
+              transition="all 0.25s ease-in-out"
+              _hover={{
```
Comment (evilpeach) [inline src/lib/layout/AppMenu.tsx:85]: I think we should change cursor while hovering here.

File: src/lib/layout/mobile/NavDrawer.tsx [hunk 4]
```diff
@@ -77,7 +77,13 @@ export const NavDrawer = () => {
 
   return (
     <>
-      <Button variant="outline-gray" size="sm" gap={1} onClick={() => onOpen()}>
+      <Button
+        variant="ghost-gray"
+        h="full"
+        borderRadius={0}
+        gap={1}
+        onClick={() => onOpen()}
```
Comment (evilpeach) [inline src/lib/layout/mobile/NavDrawer.tsx:85]: onClick={onOpen}

File: src/lib/layout/Header.tsx [hunk 5]
```diff
@@ -1,43 +1,93 @@
...
+import { useCelatoneApp, useInitia } from "lib/app-provider";
 import { AppLink } from "lib/components/AppLink";
 import { FaucetBtn } from "lib/components/button";
 import { WalletSection } from "lib/components/Wallet";
 
+import { AppMenu } from "./AppMenu";
 import { NetworkMenu } from "./NetworkMenu";
 import Searchbar from "./Searchbar";
 
+export const SectionWrapper = ({
```
Comment (evilpeach) [inline src/lib/layout/Header.tsx:13]: Create file for this component and don't forget to declare `SectionWrapperProps`

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]
LGTM

**Pull request merged**